laurent22/joplin
# Desktop: Fixes #8937: Preserve previous values when updating a note's date fields

The date fields would not display their previous value when attempting to update the "Created" and "Updated" fields on a note, via the properties.
- This was a minor usability issue, and made it harder for users to update the field.

This was previously working up until https://github.com/laurent22/joplin/pull/7243 where react-datetime was updated to v3. As per the [React-datetime v3 release notes](https://github.com/arqex/react-datetime/releases/tag/v3.0.1)
 it was caused specifically by a change in a prop name 
> - defaultValue prop is now called initialValue.

This only affects one line in the code base.

## Verification

Previously:
![image](https://github.com/laurent22/joplin/assets/32354598/b6a307bf-140b-491a-869c-d45fd4346724)

Currently:
![image](https://github.com/laurent22/joplin/assets/32354598/de04deee-83ba-41ff-89af-08d89044690f)

## How to test
1. Create a note
2. Go to note properties
3. Attempt to edit 'Created' & 'Updated' fields
4. See that the previously displayed value is available as text to edit


Fixes #8937
